packages/protobuf/src/reflect/reflect-types.ts

```diff
@@ -264,12 +264,18 @@ export interface ReflectMap<K extends MapEntryKey = MapEntryKey, V = unknown>
   [unsafeLocal]: Record<string, unknown>;
 }
 
+/**
+ * A ReflectMap key.
+ */
 export type MapEntryKey = string | number | bigint | boolean;
 
 type enumVal = number;
 
+/**
+ * The return type of ReflectMessage.get()
+ */
 // prettier-ignore
-type ReflectGetValue<Field extends DescField = DescField> = (
+export type ReflectGetValue<Field extends DescField = DescField> = (
```

Comment:
We can't infer all types with a class implementation, so we have to export this type. Another disadvantage is that classes are more verbose, hence the small bump in bundle size.